DEVELOPER_GUIDE.md

```diff
@@ -74,6 +84,74 @@ This is typically created in PyCharm IDE by visiting the `Python Interpreter`, s
 `
 In order to run tests within the PyCharm IDE, ensure the `Python Integrated Tools` / `Testing` / `Default Test Runner` is set to `pytest`.
 
+## Setting Up a Local OpenSearch Cluster For OSB Development (Optional)
+
+### OpenSearch Installation  
+
+Download the latest release of OpenSearch from https://opensearch.org/downloads.html. If you are using WSL, make sure to download it into your `/home/<user>` directory instead of `/mnt/c`. 
+```
+wget https://artifacts.opensearch.org/releases/bundle/opensearch/<x.x.x>/opensearch-<x.x.x>-linux-x64.tar.gz
+tar -xf opensearch-x.x.x-linux-x64.tar.gz
+cd opensearch-x.x.x
+```
```

Comment:
Some of this material is being updated in the doc repo https://github.com/opensearch-project/documentation-website.  It would be good to consolidate the two and direct readers there.

Reply:
I just saw your [PR](https://github.com/opensearch-project/documentation-website/pull/5177). Shall I remove this section and just redirect to that page once the PR is merged?

Reply:
@AkshathRaghav Yes, let's remove this section and redirect it to the page once that PR is merged in. We will have the other PR merged in shortly.